docs/operational_guide/upgrading_m3.md

```diff
@@ -0,0 +1,73 @@
+# Upgrading M3
+
+## Overview
+
+This guide explains how to upgrade (and downgrade??) M3 from one version to another (e.g. from 0.14.0 to 0.15.0) - both on Kubernetes as well as on bare metal.
+This includes upgrading:
+
+- m3dbnode
+- m3coordinator (coming soon!)
+- m3query (coming soon!)
+- m3aggregator (coming soon!)
+
+## m3dbnode
+
+### Graphs to monitor
+
+While upgrading M3DB nodes, it's important to monitor the status of bootstrapping the individual nodes. This can be monitored using the [M3DB Node Details](https://grafana.com/grafana/dashboards/8126) graph.
+
+### Non-Kubernetes
+
+It is very important that for each replica set, only one node gets upgraded at a time. However, multiple nodes can be upgraded across replica sets. 
+
+1) Download new binary (linux example below).
+
+```bash
+wget "https://github.com/m3db/m3/releases/download/v0.15.0-rc.0/m3_0.15.0-rc.0_linux_amd64.tar.gz" && tar xvzf m3_0.15.0-rc.0_linux_amd64.tar.gz && rm m3_0.15.0-rc.0_linux_amd64.tar.gz
+```
+
+2) Stop and upgrade one M3DB node at a time per replica set.
+
+```bash
+# locate m3dbnode process
+ps aux | grep m3dbnode
+
+# kill and restart m3dbnode with new binary
+kill -9 <m3dbnode_pid> && ./m3_0.15.0-rc.0_linux_amd64/m3dbnode -f <config-name.yml>
```

Comment:
`kill -9` will ungracefully terminate the process and may result in data corruption. Let's mention that we recommend folks use the [systemd unit] and provide systemd instructions, then mention as an aside if not using systemd to `kill` (no `-9`) the process and wait for it to stop.

[systemd unit]: https://github.com/m3db/m3/blob/a1ccb6bc53e7b5962449434442f70125650400fe/integrations/systemd/m3dbnode.service

Reply:
Also you can use `pkill m3dbnode` instead of `ps aux | grep ...` and `kill $PID`